src/run_validation.py

```diff
@@ -10,19 +10,31 @@
 
 from system import console
 from validation_workflow.validation_args import ValidationArgs
+from validation_workflow.validation_test_runner import ValidationTestRunner  # type: ignore
 
 
 def main() -> int:
-
     args = ValidationArgs()
 
     console.configure(level=args.logging_level)
     logging.getLogger("urllib3").setLevel(logging.WARNING)
 
-    dist = args.DISTRIBUTION_MAP.get(args.distribution, None)
-    dist.download_artifacts(projects=args.projects, version=args.version)
-
-    return 0
+    dist = None
+    if args.distribution == 'docker':
+        dist = "docker"
+    elif args.distribution == "tar":
+        dist = "tar"
+    elif args.distribution == "yum":
+        dist = "yum"
+    elif args.distribution == "rpm":
+        dist = "rpm"
+
+    test_result = ValidationTestRunner.dispatch(args, dist).run()
+
+    if test_result:
+        return True
+    else:
+        return False
```

Comment:
Can we just `return test_result` here?

Reply:
Thanks for the feedback, @zelinh ...yes, we should just return test_result directly as it will be either True or False.